scripts/pancancer_classifier.py

```diff
@@ -303,31 +308,33 @@
 plt.savefig(full_roc_file, dpi=600, bbox_inches='tight')
 plt.close()
 
-# Tissue specific performance
-tissue_metrics = {}
-for tissue in tissues:
-    sample_sub = y_matrix[y_matrix.cohort == tissue].index.values
-
-    y_tissue_train = y_train[y_train.index.isin(sample_sub)]
-    y_tissue_pred_train = y_pred_train[y_train.index.isin(sample_sub)]
-    y_tissue_test = y_test[y_test.index.isin(sample_sub)]
-    y_tissue_pred_test = y_pred_test[y_test.index.isin(sample_sub)]
-    y_tissue_pred_cv = y_cv[y_train.index.isin(sample_sub)]
-
-    metrics_train_tis = get_threshold_metrics(y_tissue_train,
-                                              y_tissue_pred_train,
-                                              tissue=tissue)
-    metrics_test_tis = get_threshold_metrics(y_tissue_test, y_tissue_pred_test,
-                                             tissue=tissue)
-    metrics_cv_tis = get_threshold_metrics(y_tissue_train, y_tissue_pred_cv,
-                                           tissue=tissue)
-    tissue_metrics[tissue] = [metrics_train_tis, metrics_test_tis,
-                              metrics_cv_tis]
-
-tissue_auroc = {}
-for tissue, metrics_val in tissue_metrics.items():
+# disease specific performance
+disease_metrics = {}
+for disease in diseases:
+    sample_sub = y_matrix[y_matrix.cohort == disease].index.values
+
+    y_disease_train = y_train[y_train.index.isin(sample_sub)]
+    y_disease_pred_train = y_pred_train[y_train.index.isin(sample_sub)]
```

Comment:
`y_train` is the true labels, `y_pred_train` are the predicted labels. I'll add a comment and tweak the variable names a bit to make this more clear